notebooks/cugraph_benchmarks/nx_cugraph_bc_benchmarking.ipynb

```diff
@@ -0,0 +1,221 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Benchmarking NetworkX compatibility\n",
+    "This notebook benchmark the use of a NetworkX Graph object as input into algorithms.  <p>\n",
+    "The intention of the feature is to be able to drop cuGraph into existing NetworkX code in spot where performance is not optimal.\n",
+    "\n",
+    "\n",
+    "### Betweenness Centrality\n",
+    "Both NetworkX and cuGraph allow for estimating the betweenness centrality score by using a subset of vertices rather than all the vertices.  WHile that does produce a less accurate answer, it dramatically improves performance when the sample is small.  For this test, the algorithms will use only 10% of the vertices to compute the estimate \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "__Notebook Credits__\n",
+    "\n",
+    "* Original Authors: Bradley Rees\n",
+    "* Last Edit: 09/27/2020\n",
+    "\n",
+    "RAPIDS Versions: 0.16\n",
+    "\n",
+    "Test Hardware\n",
+    "```\n",
+    "    GV100 32G, CUDA 10,0\n",
+    "    Intel(R) Core(TM) CPU i7-7800X @ 3.50GHz\n",
+    "    32GB system memory\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import networkx as nx\n",
+    "import cugraph as cudanx\n",
```

Comment:
cudanx :) ?

Reply:
One suggestion was that we load cuGraph as "cuda.nx"  then the update would read as "cuda.nx.<algo> but python does not allow "dots" in "as" names.  The suggestion is to then do
      import cugraph as cnx
That makes for a single letter change to code

Reply:
updated in notebook to cnx